distribution/docs/src/main/resources/_contents/_configuring/configuring-from-the-admin-console-contents.adoc

```diff
@@ -0,0 +1,156 @@
+====
+
+==== Viewing Currently Active Applications from ${admin-console}
+
+${branding} displays all active applications in the ${admin-console}.
+This view can be configured according to preference.
+Either view has an `>` arrow icon to view more information about the application as currently configured.
+
+.${admin-console} Views
+[cols="1,4", options="header"]
+|===
+|View
+|Description
+
+|Tile View
+|The first view presented is the Tile View, displaying all active applications as individual tiles.
+
+|List View
+|Optionally, active applications can be displayed in a list format by clicking the list view button.
+
+|===
+
+===== Application Detailed View
+
+Each individual application has a detailed view to view information specific to that application, adjust configurations or enable/disable features.
+All applications have a standard set of tabs, although some apps may have additional ones with further information.
+
+.Individual Application Views
+[cols="1,4", options="header"]
+|===
+|Configuration
+|The Configuration tab lists all bundles associated with the application as links to configure any configurable properties of that bundle.
+
+|Details
+|The Details tab gives a description, version, status, and list of other applications that either required for, or rely on, the current application.
+
+|Features
+|The features tab breaks down the individual features of the application that can be installed or uninstalled as configurable features.
+
+|===
+
+==== Managing Applications
+
+The *Manage* button enables activation/deactivation and adding/removing applications.
+
+===== Activating / Deactivating Applications
+
+The *Deactivate* button stops individual applications and any dependent apps.
+Certain applications are central to overall functionality and cannot be deactivated.
+These will have the *Deactivate* button disabled.
+Disabled apps will be moved to a list at the bottom of the page, with an enable button to reactivate, if desired.
+
+[IMPORTANT]
+====
+Deactivating the `platform-app`, `admin-app`, and `security-services-app` will cause errors within the system, so the capabilities to do so have been DISABLED.
+====
+
+===== Adding Applications
+
+The *Add Application* button is at the end of the list of currently active applications.
+
+===== Removing Applications
+
+To remove an application, it must first be deactivated.
+This enables the *Remove Application* button.
+
+===== Upgrading Applications
+
+Each application tile includes an *Upgrade* button to select a new version to install.
+
+==== System Settings Tab
+
+The configuration and features installed can be viewed and edited from the System tab as well, however, it is recommended that configuration be managed from the applications tab.
+
+[IMPORTANT]
+====
+In general, applications should be managed via the applications tab.
+Configuration via this page could result in an unstable system.
+Proceed with caution!
+====
+
+===== Managing Features Using the ${admin-console}
```

Comment:
Looks like this is the information I was looking for regarding feature management. Not sure it is a good fit for the `System Settings Tab` section since it talks about specific applications. Should we move it to 3.1.3 or make it its own top-level section?